doc/tutorial/basic/index.md

```diff
@@ -71,27 +72,19 @@ object TutorialApp {
 As you expect, this will simply print "HelloWorld" when run. To run this, simply launch `sbt` and invoke the `run` task:
 
     $ sbt
-    > run
-    [info] Compiling 1 Scala source to (...)/scala-js-tutorial/target/scala-2.12/classes...
-    [info] Fast optimizing (...)/scalajs-tutorial/target/scala-2.12/scala-js-tutorial-fastopt.js
+    sbt:Scala.js Tutorial> run
+    [info] Compiling 1 Scala source to (...)/scala-js-tutorial/target/scala-2.13/classes ...
+    [info] Fast optimizing (...)/scala-js-tutorial/target/scala-2.13/scala-js-tutorial-fastopt.js
     [info] Running tutorial.webapp.TutorialApp
     Hello world!
     [success] (...)
 
-Congratulations! You have successfully compiled and run your first Scala.js application. The code is actually run by a JavaScript interpreter. If you do not believe this (it happens to us occasionally), you can use the `last` command in sbt:
-
-    > last
-    (...)
-    [info] Running tutorial.webapp.TutorialApp
-    [debug] with JSEnv ExternalJSEnv for Node.js
-    [debug] Starting process: node
-    [success] (...)
-
-So your code has actually been executed by Node.js.
```

Comment:
Why was this section removed? At least this deserves mentioning in the commit message IMO.

Reply:
Yes, it was removed because that [debug] message doesn't appear anymore, so it's all pointless.

Reply:
Ah :P